docs/basic_training/index.es.md

```diff
@@ -0,0 +1,31 @@
+---
+description: Descripción general del material de formación básico de Nextflow
+hide:
+    - toc
+---
+
+# Bienvenido
+
+## Bienvenido al taller de capacitación de Nextflow
+
+Nos complace acompañarlo en el camino para escribir flujos de trabajo científicos reproducibles y escalables usando Nextflow. Esta guía complementa la documentación completa de Nextflow; si alguna vez tiene alguna duda, diríjase a los documentos que se encuentran [aquí] (https://www.nextflow.io/docs/latest).
```

Comment:
this section  [aquí] (https://www.nextflow.io/docs/latest) is not rendering properly